lib/split/alternative.rb

```diff
@@ -5,8 +5,9 @@ class Alternative
     attr_accessor :experiment_name
     attr_accessor :weight
     attr_accessor :recorded_info
+    attr_accessor :version
 
-    def initialize(name, experiment_name)
+    def initialize(name, experiment_name, version = 0)
```

Comment:
After this change, does a `Split::Alternative` represent a single Alternative, independent of version, or a single version of an Alternative?

Reply:
Example where this probably matters:

There's a delete method (Github won't let me comment on it :-/)
```ruby
     def delete
       Split.redis.del(key)
     end
```

Before this change, callers could assume after this method was called that the `Split::Alternative` no longer exists (in Redis). However, after this change the `delete` method will only delete the version that was passed in when the object was created. This change in semantics could break some other piece of code that was relying on the desired behavior.

Reply:
agreed, should probably disable deletion.